resources/home/dnanexus/eggd_artemis.py

```
@@ -145,6 +147,42 @@ def main(
+    nmd_samples = []
+    nmd_panels = []
+
+    # setting up defaults - in some of the NMD checks, we want to replace missing data with empty data.
+    # We're only concerned with the count in the instance of SNVs
+    default_snv_report = [{"SNV count": 0}]
+    # The CNV report has only two entries that we check: the CNV count, and dataframe of missing regions.
+    # The "default" is a dataframe with only one row - the column names. One of our checks is to look for instances where len(df) <= 1
+    indices = ["a", "b", "c", "d", "e", "f", "g", "h", "i"]
+    columns = ["CNV excluded regions", "Chrom", "Start", "End", "Length", "Gene_Symbol", "HGNC_ID", "Transcript", "Exon"]
+    excluded_regions = [{index: column} for index, column in zip(indices, columns)]
+    default_cnv_report = [{"CNV count": 0, "cnv_excluded_regions_df": pd.DataFrame(excluded_regions)}]
```

Ruff (0.14.1)

Undefined name `pd`

(F821)